src/AbstractFactory/ConfigAbstractFactory.php

```diff
@@ -64,8 +72,8 @@ public function __invoke(\Interop\Container\ContainerInterface $container, $requ
 
         $serviceDependencies = $dependencies[$requestedName];
 
-        if ($serviceDependencies !== array_values(array_map('strval', $serviceDependencies))) {
-            $problem = json_encode(array_map('gettype', $serviceDependencies));
+        if ($serviceDependencies !== array_values(array_map('\strval', $serviceDependencies))) {
```

Comment:
Functions and classes referenced inside of strings are always considered globally qualified; there's no reason to prefix them.